src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from 'react';
+import { PeopleFilters } from './PeopleFilters';
+import { Loader } from '../../components/Loader';
+import { PeopleTable } from './PeopleTable';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
```

Comment:
Would be better if you store null when you don't have the initial value & want to fetch params from server
```suggestion
  const [people, setPeople] = useState<Person[] | null>(null);
```